packit_service/worker/checker/distgit.py

```diff
@@ -128,6 +128,30 @@ def pre_check(self) -> bool:
         return True
 
 
+class PermissionOnDistgitForFedoraCI(Checker):
+    def pre_check(self) -> bool:
+        if self.data.event_type in (pagure.pr.Comment.event_type(),):
+            comment = self.data.event_dict.get("comment", "")
+            commands = get_packit_commands_from_comment(
+                comment,
+                self.service_config.comment_command_prefix,
+            )
+            command = commands[0] if commands else ""
+            commenter = self.data.actor
+            logger.debug(f'Triggering CI through "{command}" comment by: {commenter}')
+            if not self.is_packager(commenter):
+                msg = (
+                    "CI retriggering "
```

Comment:
not sure how we do it for downstream automation (for consistency), but it would be beneficial to comment this also in the PR, WDYT?

Reply:
For downstream automation we do the "task was accepted" comments, we can do that here as well, though I have a feeling certain users would consider that unnecessary spam :slightly_smiling_face: But I suppose we can make that configurable later.

Reply:
sorry for not being specific, I meant mostly error-related comments

but yes, the `Task was accepted` could be quite spammy, so not sure whether to do this by default, the reactions UX would be much better here, so that could be used once the Forgejo switch happens

Reply:
> sorry for not being specific, I meant mostly error-related comments

Oh, right, for downstream automation errors are reported as issues in `issue_repository` if configured. For CI I think it makes sense to reply with a comment. Let me do that.

> but yes, the `Task was accepted` could be quite spammy, so not sure whether to do this by default, the reactions UX would be much better here, so that could be used once the Forgejo switch happens

There is a TODO for it, perhaps I can implement it and we can disable it later?
https://github.com/packit/packit-service/blob/615f072d16e8f4e57eb3ae3f9b6191262159ceea/packit_service/worker/jobs.py#L456

Reply:
> For CI I think it makes sense to reply with a comment. Let me do that.

👍 , thanks!

> There is a TODO for it, perhaps I can implement it and we can disable it later?

actually, thinking about it now, for Fedora CI, operating on PRs, it makes sense to do regular commit statuses exactly like in upstream

Reply:
> actually, thinking about it now, for Fedora CI, operating on PRs, it makes sense to do regular commit statuses exactly like in upstream

Do you mean the flags (as Pagure calls them) that are already there so no extra reporting needed or did I misunderstood?

Reply:
yes I mean the flags, but for Fedora CI we haven't been doing them for `Task was accepted` yet iiuc

Reply:
Oh, so you mean another status before `RPM build was submitted ...`?

Reply:
yes

Reply:
Ok, makes sense, let me to that.